### PR TITLE
Refactor headers and footers

### LINKED ISSUE #82: The "Derive a Secret" menu isn't triggered by the icons in iOS
In the DiceKeyPresent page, in the footer object, there is a DerivationRecipeMenu created with icons in the label.

Click the icons does not open the menu, as it should.

Rather, to open your finger needs to hit the tiny section of text at the bottom.

### PR DESCRIPTION
**Pull request by UppaJung**
fixes #82

introduces some styling differences

### CODE REVIEW

File: DiceKeys/Views/Screens/DiceKeyPresent.swift [hunk 1]
```diff
@@ -26,49 +26,56 @@ struct DiceKeyPresentNavigationFooter: View {
...
+    
+    var navBarContentHeight: CGFloat { geometry.size.height / 12 }
+    var navBarImageHeight: CGFloat { navBarContentHeight * 0.75 }
 
     var body: some View {
         let view = VStack {
         ZStack {
             HStack(alignment: .top) {
-                
+                Spacer()
```
Comment (Abhityagi16) [inline DiceKeys/Views/Screens/DiceKeyPresent.swift:37]: Why do we need this spacer? Let the icon take all of left space

File: DiceKeys/Views/Screens/DiceKeyPresent.swift [hunk 2]
```diff
@@ -26,49 +26,56 @@ struct DiceKeyPresentNavigationFooter: View {
...
-                            Spacer()
-                        }
-                    }
+                        Spacer(minLength: 2)
+                        Text("Derive a Secret").multilineTextAlignment(.center).font(.footnote)
+                    }.frame(width: geometry.size.width * BottomButtonFractionalWidth,
+                            height: navBarContentHeight)
                 }
-                .frame(width: geometry.size.width * BottomButtonFractionalWidth, alignment: .center)
+                .frame(width: geometry.size.width * BottomButtonFractionalWidth,
```
Comment (Abhityagi16) [inline DiceKeys/Views/Screens/DiceKeyPresent.swift:74]: We probably don't need to set both frames. Remove the frame in line 71 if we keep this one.

File: DiceKeys/Views/Screens/DiceKeyPresent.swift (hunk 1 above)
Comment (UppaJung) [reply DiceKeys/Views/Screens/DiceKeyPresent.swift:37]: The footer is outside the icons.  It ensures there's space on the left edge of the footer and the right edge of the footer.

File: DiceKeys/Views/Screens/DiceKeyPresent.swift (hunk 2 above)
Comment (UppaJung) [reply DiceKeys/Views/Screens/DiceKeyPresent.swift:69]: Fixed.

**Review by Abhityagi16 — APPROVED** [review]

**Pull request merged**